src/webVitals.ts

```diff
@@ -10,7 +12,7 @@ export function withAxiom(nextConfig: NextConfig): NextConfig {
     rewrites: async () => {
       const rewrites = await nextConfig.rewrites?.();
 
-      const ingestEndpoint = process.env.AXIOM_INGEST_ENDPOINT || process.env.NEXT_PUBLIC_AXIOM_INGEST_ENDPOINT;
+      const ingestEndpoint = getIngestURL(EndpointType.webVitals);
```

Comment:
withAxiom should live in `index.ts` and configure a rewrite for both web-vitals and logs.